### PR TITLE
[FIX] Directory and Apps logs page

### LINKED ISSUE #13909: Directory fields wrong size
As seen below when Federation is enabled:

image

### PR DESCRIPTION
**Pull request by tassoevan**
Closes #13909

### CODE REVIEW

**Comment by sampaiodiego** [comment]
seems apps are using `rc-directory` class [here](https://github.com/RocketChat/Rocket.Chat/blob/develop/app/apps/client/admin/apps.html#L10) .. it should be broken right?

**Comment by tassoevan** [comment]
@sampaiodiego Awkward style reuse... Thanks for warning me.

**Comment by sudipt1999** [comment]
@tassoevan  How css gets broken up here ! In my local environment I am could fix it by using 100% height to input field but it doesn't fix on @sampaiodiego .

File: app/apps/client/admin/apps.html [hunk 1]
```diff
@@ -1,20 +1,34 @@
...
+						{{/if}}
 					</div>
-				</label>
-			</div>
+					<input
+						type="text"
+						class="rc-input__element js-search"
+						name="message-search"
+						id="message-search"
+						placeholder="{{_ "Search_Apps"}}"
```
Comment (sampaiodiego) [inline app/apps/client/admin/apps.html]: `Search_Apps` is missing from i18n files

**Comment by rodrigok** [comment]
@tassoevan can you fix the margins of the app's logs in this PR?

**Pull request merged**